test/Streamly/Test/Common/Array.hs

```diff
@@ -86,6 +85,23 @@ testLengthFromStreamN = genericTestFrom A.fromStreamN
 #ifndef TEST_SMALL_ARRAY
 testLengthFromStream :: Property
 testLengthFromStream = genericTestFrom (const A.fromStream)
+
+testFromListN :: Property
+testFromListN =
+    forAll (choose (0, maxArrLen)) $ \len ->
+        forAll (choose (0, len)) $ \n ->
+            forAll (vectorOf len (arbitrary :: Gen Int)) $ \list ->
+                monadicIO $ do
+                    let arr = A.fromListN n list
+                    assert (A.toList arr == take n list)
+
+testFromList :: Property
```

Comment:
The above comment is applicable here as well.

Reply:
This is applicable for #ifndef TEST_SMALL_ARRAY